packages/c/cello/xmake.lua

```diff
@@ -0,0 +1,53 @@
+package("cello")
+    set_homepage("http://libcello.org/")
+    set_description("Higher level programming in C")
+
+    add_urls("https://github.com/orangeduck/Cello/archive/refs/tags/v$(version).tar.gz",
+             "https://github.com/orangeduck/Cello.git")
+
+    add_versions("2.0.3", "1afcae06f5efc10ea161737a862073ff5679c964540bca7cd719539609d0633c")
+
+    on_install("@!windows", "android@linux", function(package)
+        io.writefile("xmake.lua", [[
+            add_rules("mode.release", "mode.debug")
+            target("Cello")
+                set_kind("$(kind)")
+                set_languages("gnu99")
+                add_links("m", "pthread")
+                add_files("src/*.c")
+                add_headerfiles("include/Cello.h")
+                add_includedirs("include")
+                on_config(function(target)
+                    if is_plat("@mingw") and target:has_cincludes("Dbghelp.h") then
+                        target:add("links", "Dbghelp")
+                    elseif is_plat("@linux") and target:has_cincludes("execinfo.h") then
+                        target:add({ links = "execinfo", ldflags = "-rdynamic" })
```

Comment:
and here

Reply:
Resolved in [ef9dca4](https://github.com/xmake-io/xmake-repo/pull/3679/commits/ef9dca451d67d2f176bbcfe5edd0920e6d711786).